### PR TITLE
add search by day and concelho, add active v2 api

### PR DESCRIPTION
**Pull request by tomahock**

### CODE REVIEW

File: app/Http/Controllers/IncidentController.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            'success' => true,
+            'data' => IncidentResource::collection($incidents),
+        ]);
+    }
+
+    public function search(Request $request): JsonResponse
+    {
+        //@todo validate date format
+        $day = $request->get('day');
+        if(!$day){
```
Comment (josepostiga) [inline app/Http/Controllers/IncidentController.php:35]: It would probably be interesting to use Form Requests to validate all required data, no?

**Review by josepostiga — APPROVED** [review]
Hey, this looks good. If you have the time to look at my notes, we could make this a little better. If you need help, ping me in Slack.

File: app/Http/Controllers/IncidentController.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        }
+
+        $date = new Carbon($day);
+        $all = $request->get('all');
+        $concelho = $request->get('concelho');
+
+        $incidents = Incident::whereBetween(
+                'dateTime',
+                array(
+                    Carbon::parse($date->startOfDay()),
```
[the remark this answers is not in the document]
Comment (tomahock) [reply app/Http/Controllers/IncidentController.php:46]: If i take the `parse()` I don't have results! Keeping it for now :)

**Comment by tomahock** [comment]
@josepostiga changes made from your feedback. Should be perfect now :D

**Review by josepostiga — APPROVED** [review]
Thank you for the additional effort taken! :clap:

**Pull request merged**